.github/workflows/core-profile-tck-ee11-snapshot.yml

```diff
@@ -0,0 +1,83 @@
+# This is a basic workflow that is manually triggered
```

Comment:
This isn't accurate.

Reply:
Yes, I'll remove that. My initial intention was to make it manual only, but then I wanted to see it run so I made it part of the PR.